README.md

```diff
@@ -334,11 +334,13 @@ import { ThemeableMixin, theme } from '@dojo/widget-core/mixins/Themeable';
 @theme(css)
 export default class MyWidget extends ThemeableMixin(WidgetBase) {
     protected render() {
-        return v('div', { classes: this.classes(css.root).fixed(css.rootFixed) });
+        return v('div', { classes: [ this.theme(css.root), css.rootFixed ] });
```

Comment:
Isn't this supposed to be:

```diff
-return v('div', { classes: [ this.theme(css.root), css.rootFixed ] })
+return v('div', { classes: [ ...this.theme(css.root), css.rootFixed ] })
```

Reply:
Nevermind... I see below about it... it does feel a bit confusing without an example maybe actually showing a code example after the return array and spreading it would be useful.  Also, IIRC from the changes, `this.theme()` only accepts a single argument?  If that is correct it might be better to be a bit more specific above when we say _`this.theme()` that consumes string class names and returns the theme's equivalent class names._ to something like _`this.theme()` takes a single argument that is either a string class name or an array of string class names and returns the theme's equivalent class names as either a single string or array of strings?_